build.gradle

```diff
@@ -25,31 +25,38 @@ import org.apache.tools.ant.filters.ReplaceTokens
 
 buildscript {
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        gradlePluginPortal()
+        mavenCentral()
     }
 
     dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:6.1.0'
         classpath "io.github.slimjar:gradle-plugin:1.3.0"
     }
 }
 
+plugins {
+    id 'com.github.johnrengelman.shadow' version '8.1.1'
+    id 'java'
+}
+
 ext {
     libsBase = 'me.lorenzo0111.elections.libs'
 }
 
+sourceCompatibility = '1.17'
+targetCompatibility = '1.17'
 
 subprojects {
     apply plugin: 'java'
     apply plugin: 'com.github.johnrengelman.shadow'
     apply plugin: "io.github.slimjar"
 
-    sourceCompatibility = '1.8'
-    targetCompatibility = '1.8'
+    sourceCompatibility = '1.17'
+    targetCompatibility = '1.17'
     compileJava.options.encoding = "UTF-8"
     compileTestJava.options.encoding = "UTF-8"
 
-    version = "1.6"
+    version = "1.6.mcplaydates-1.0"
```

Comment:
Are you able to remove this part?

Reply:
Yep, will do

Reply:
fixed in commit 1f2e3cd359e4e1f9070942b4adffb935f1f48b03